ec2/spark_ec2.py

```diff
@@ -569,6 +569,8 @@ def launch_cluster(conn, opts, cluster_name):
         master_nodes = master_res.instances
         print "Launched master in %s, regid = %s" % (zone, master_res.id)
 
+    # Wait for the information of the just-launched instances to be propagated within AWS
+    time.sleep(5)
```

Comment:
@nchammas 
Maybe we should insert a print here to tell the client that we are waiting for the information to be propagated, since there will be 5 seconds idle between `launch master in` and `wait for the cluster to enter`. 
How do you think?